integration/solid/dyn_linesearch_solve/dyn_linesearch_solve.ats

```diff
@@ -9,6 +9,6 @@ tolerance_test(name="dyn_linesearch_solve_serial",
     input_file="{0}/tests/solid/dyn_linesearch_solve.lua".format(serac_input_files_dir))
 
 tolerance_test(name="dyn_linesearch_solve_parallel",
-    input_file="{0}/tests/solid/dyn_linesearch_solve.lua".format(serac_input_files_dir),
+    input_file="{0}/tests/solid/dyn_linesearch_solve.lua".format(serac_input_files_dir)
```

Comment:
These are missing a comma at the end.

Reply:
yup i just noticed that and fixed it now